React/Modules/RCTDevMenu.m

```diff
@@ -185,7 +185,7 @@ - (instancetype)init
       [weakSelf.bridge.eventDispatcher sendDeviceEventWithName:@"toggleElementInspector" body:nil];
     }]];
 
-    _webSocketExecutorName = [_defaults objectForKey:@"websocket-executor-name"] ?: @"Chrome";
+    _webSocketExecutorName = [_defaults objectForKey:@"websocket-executor-name"] ?: @"JS Remotely";
```

Comment:
@mkonicek Jordan suggested that, but looks like @frantic suggested we go the debug-hub route for now. Attaching to JSC for Android may also require us to tweak the JSC .so that is shipped with ReactNative.